h/views/activity.py

```diff
@@ -349,6 +349,19 @@ def domain(user):  # pragma: no cover
             "orcid": self.user.orcid,
         }
 
+        if not self.request.user and (self.user.nipsa or annotation_count == 0):
```

Comment:
`self.request.user` is the logged in user and `self.user` is the requested profile user.